source/lib/vagrant-openstack-provider/errors.rb

```diff
@@ -51,6 +51,10 @@ class NoMatchingImage < VagrantOpenstackError
         error_key(:no_matching_image)
       end
 
+      class ConflictImageName < VagrantOpenstackError
```

Comment:
This error is never used. It should be remove (and the associated error_key). Furthermore, an error class already for that case : `ConflictBootOption`